include/dev/TPS2HB50BQ1.hpp

```diff
@@ -0,0 +1,77 @@
+// Driver for power switch
+#ifndef LVSS_TPS2HB50BQ1_HPP
+#define LVSS_TPS2HB50BQ1_HPP
+
+#include <EVT/io/ADC.hpp>
+#include <EVT/io/CAN.hpp>
+#include <EVT/io/CANOpenMacros.hpp>
+#include <EVT/io/CANopen.hpp>
+#include <EVT/io/GPIO.hpp>
+
+namespace IO = EVT::core::IO;
+
+namespace LVSS {
+
+/**
+ * Power switch for LVSS
+ * /datasheets/TPS2HB35-Q1 Power Switch Data Sheet.pdf
+ */
+class TPS2HB50BQ1 {
+public:
+    TPS2HB50BQ1(IO::GPIO& en1, IO::GPIO& en2, IO::GPIO& senseOut, IO::GPIO& latch, IO::GPIO& diagEn,
+                IO::GPIO& diagSelect1, IO::GPIO& diagSelect2, IO::ADC& adc, uint8_t current_diag_mode);
+    void enableAll();
+
+    void disableAll();
+
+    void enable1();
+
+    void disable1();
+
+    void enable2();
+
+    void disable2();
```

Comment:
I would recommend spelling out the numbers. Helps with readability and makes it easier to catch a bug if you call a function with the wrong number. More of a nitpick though